### PR TITLE
Task10

### PR DESCRIPTION
**Pull request by Julian22222**
completed task 10

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -367,3 +366,161 @@ test("status:400, Handling psql errror for /api/articles/banana/comments, return
...
+//               body: expect.any(String),
+//               created_at: expect.any(String),
+//               votes: expect.any(Number),
+//               comment_count: expect.any(Number),
+//             })
+//           );
+//         });
+//       });
+//   });
+// });
```
Comment (Angaca) [inline __tests__/app.test.js:526]: No graveyard of code please!

File: app.js [hunk 2]
```diff
@@ -22,9 +25,15 @@ app.get("/api/articles", getAllArticles);
 
 app.get("/api/articles/:article_id/comments", getComments);
 
+app.post("/api/articles/:article_id/comments", postComments);
+
 app.use((err, req, res, next) => {
   if (err.code === "22P02") {
     res.status(400).send({ msg: "Bad request, invalid article id" });
+  } else if (err.code === "23503") {
+    res.status(404).send({ msg: "Not Found" });
```
Comment (Angaca) [inline app.js:34]: Perhaps this error message can be a bit more user-friendly and descriptive?

**Review by Angaca — APPROVED** [review]
The function itself is good but please don't send another pull request with all this code commented out, either you fix your code or that shouldn't be there. This repo must be clean before you merge please

**Comment by Angaca** [comment]
And please your commit messages should express what are they doing

File: __tests__/app.test.js (hunk 1 above)
Comment (Julian22222) [reply __tests__/app.test.js:526]: resolved it

**Pull request merged**